app/js/Button.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export default class Button extends React.PureComponent {
```

Comment:
I don't think we should add react-bootstrap at this time actually. It's a large library, and I don't think we'd see much benefit. I linked its docs because I thought it was a decent approach to integrating React+Bootstrap. I'm happy to stick with your `Button` component or go back to getting the correct class names.